django_irods/views.py

```diff
@@ -174,6 +171,8 @@ def download(request, path, use_async=True, use_reverse_proxy=True,
             delete_zip.apply_async((irods_output_path,),
                                    countdown=(60 * 60 * 24))  # delete after 24 hours
             if api_request:
+                get_or_create_task_notification(task_id, name='zip download', payload=download_path,
+                                                username=user_id)
```

Comment:
maybe refactor this if/else
the else has the exact same `get_or_create_task_notification(task_id, name='zip download', payload=download_path, username=user_id)` so could remove that duplication

Reply:
Good catch/suggestion!